package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/handlebars-v4",
-  "version": "4.2.1",
+  "version": "4.5.2",
```

Comment:
Is there a reason why we are matching this to handlebars package version ?

Reply:
I get this is a wrapper but since we publish this to npm I would love if we could follow semver

Reply:
Just makes it easier to consume when we know what version we are wrapping.

Reply:
As for semver, it makes sense that we would mirror the underlying version since whatever changes happen there also happen to this package.